addons/web/static/src/search/control_panel/control_panel.scss

```diff
@@ -24,12 +24,14 @@
 
             .o_control_panel_actions {
                 min-width: MIN(500px, 33%);
+                margin: 0 2%;
```

Comment:
What's the goal of this margin ?

Reply:
https://www.awesomescreenshot.com/image/50537196?key=a8c937540f1ed85e1652075d34779408

Reply:
`text-truncate` added back, the margin is no longer needed.